backend/tests/fixture/LinearMembers.ts

```diff
@@ -0,0 +1,21 @@
+export const linearMember = {
+  nodes: [
+    {
+      active: true,
+      admin: false,
+      archivedAt: undefined,
+      avatarUrl: undefined,
+      createdAt: "2022-01-05T02:55:21.670Z",
+      createdIssueCount: 3,
+      disableReason: undefined,
+      displayName: "ravi",
+      email: "ravi@linktr.ee",
```

Comment:
I'm not sure this fixture has removed all sensitive data, please double check

Reply:
though its not sensitive infos, still modified it to make it look less sensitive.